earth_osm/utils.py

```diff
@@ -131,7 +131,34 @@ def convert_pd_to_gdf_lines(df_way):
     gdf.drop(columns=["lonlat"], inplace=True)
     return gdf
 
-def output_csv_geojson(df_feature, primary_name, feature_name, region_list, data_dir):
+
+def write_csv(df_feature, outputfile_partial, feature_name, out_aggregate, fn_name):
+    """Create csv file. Optimized for large files as write on disk in chunks"""
+    if out_aggregate:
+        output_path = os.path.join(outputfile_partial, f"_{feature_name}s" + ".csv")
+        df_feature.to_csv(
+            output_path, index=False, header= not os.exists(output_path), mode="a"
+        )  # Generate CSV
+    else:
+        output_path = os.path.join(outputfile_partial, f"{fn_name}_{feature_name}s" + ".csv")
+        df_feature.to_csv(output_path)  # Generate CSV
+
+
+def write_geojson(gdf_feature, outputfile_partial, feature_name, out_aggregate, fn_name):
+    """Create geojson file. Optimized for large files as write on disk in chunks"""
+    if out_aggregate:
+        output_path = os.path.join(outputfile_partial, f"all_{feature_name}s" + ".geojson")
+        gdf_feature.to_file(
+            output_path, driver="GeoJSON", index=False, mode="a"
```

Comment:
mode=("a" if not os.path.exists(......) else "w") 
this runs into a similar problem as the csv